weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ChatView/MessagePanel.tsx

```diff
@@ -22,6 +23,15 @@ export const MessagePanel = ({
   const hasToolCalls = !!message.tool_calls && message.tool_calls.length > 0;
   const hasContent = !!message.content && message.content.length > 0;
 
+  const [isShowingMore, setIsShowingMore] = useState(false);
+  const [isOverflowing, setIsOverflowing] = useState(false);
+  const contentRef = useRef<HTMLDivElement>(null);
+  useEffect(() => {
+    if (contentRef.current) {
+      setIsOverflowing(contentRef.current.scrollHeight > 400);
```

Comment:
Couldn't the scroll height change as you resize the peek drawer/window even if the content doesn't?

Reply:
this should be addressed in the other pr, as well as the not expanding the markdown
https://github.com/wandb/weave/pull/2978